crates/astria-sequencer/src/bin/app.rs

```diff
@@ -0,0 +1,18 @@
+use astria_sequencer::sequencer::Sequencer;
```

Comment:
High level comment on this: let's `mv bin/app.rs main.rs` because the main point of this crate is to provide an executable.